client/src/lib.rs

```diff
@@ -366,6 +367,34 @@ mod test {
         ));
     }
 
+    #[test]
+    fn api_key_serialization_and_parsing() {
```

Comment:
side note: this highlights some inconsistencies on our side: some activity intents expect expiration times as strings (e.g. `expiration_seconds`), whereas others expect uint64 (`session_expiry`) or even int32s (`otp_expiration_seconds, verification_token_expiration_seconds, session_expiration_seconds`). irrelevant to this change, just an observation as I was digging into some protos. 

and then on the flip side (Turnkey responses), org data stores fields like `expiration_seconds` as `uint64` for structs like API keys (which you're addressing here). 

just so I'm wrapping my head around this change as a whole correctly: we're modifying the codegen only for what parses Turnkey **responses**